runtime/Python3/src/antlr4/Recognizer.py

```diff
@@ -52,7 +52,7 @@ def removeErrorListeners(self):
     def getTokenTypeMap(self):
         tokenNames = self.getTokenNames()
         if tokenNames is None:
-            from antlr4.error.Errors import UnsupportedOperationException
+            from .error.Errors import UnsupportedOperationException
```

Comment:
I'm guessing that the reason these imports were made lazy was because of circular import issues... This MR should start to straighten that out.